### PR TITLE
Add the official router and several mixins.

### PR DESCRIPTION
**Pull request by RikuVan**
I find this repo a useful starting point for Hyperapp but it needs some updating of the mixins/libraries. I have added the official router and several mixins which are also on npm. If this is too much at once, it would be nice to at least have the official router here. Seems a bit tedious though to make each mixin a separate pr.

### CODE REVIEW

**Comment by jbucaran** [comment]
Looks good, maybe we can sort it alphabetically if that's not too much to ask?

**Comment by RikuVan** [comment]
Aren't they already sorted? Seemed logical to have the @hyperapp/** at the top. Did you mean that the '@' should be ignored and they should be sorted with the others?

**Comment by jbucaran** [comment]
@RikuVan Yes, thought it would look better that way, but separating official packages from non-official packages is also a good idea.

**Comment by RikuVan** [comment]
Here they are sorted. Also took the liberty of splitting official and non-official resources. I personally think the 'how it works' gist is great and nice to see the blog posts. But if you think they don't belong, I can remove them.

**Comment by zaceno** [comment]
@RikuVan Just wanted to say thanks for putting in the work on this PR! ❤️  I've been feeling for a while that the awesome list is out of date and missing a lot of things. 

Also thanks for including hyperapp-partial among the many excellent packages you added!

**Comment by RikuVan** [comment]
Sure, I am happy to contribute, even in this tiny way.

**Comment by zaceno** [comment]
@jbucaran merge?

File: README.md [hunk 1]
```diff
@@ -23,13 +25,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
 - [hypernews](https://github.com/traducer/hypernews) - Hacker News clone with Hyperapp.
 
 ## Libraries and Mixins
+- [hyperapp-deepupdate](https://github.com/NoobLad/hyperapp-deepupdate) - A utility for updating deeply nested state.
+- [hyperapp-dot-notation-reducer](https://github.com/alber70g/hyperapp-dot-notation-reducer) - Allows actions to return an object with a path as a path as a property.
+- [hyperapp-firebase-auth](https://github.com/lukejacksonn/hyperapp-firebase-auth) - Drop in authentication for Hyperapp using Firebase.
+- [hyperapp-freeze](https://github.com/okwolf/hyperapp-freeze) - Deep freezes the state on load in development.
+- [hyperapp-hmr](https://github.com/scrapjs/hyperapp-hmr) - Hot module replacement for Hyperapp independent of the build process.
+- [hyperapp-html](https://github.com/swizz/hyperapp-html) - Html helpers for Hyperapp.
```
Comment (tobiasbueschel) [inline README.md]: HTML

File: README.md [hunk 2]
```diff
@@ -23,13 +25,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
...
+- [hyperapp-server](https://github.com/hyperapp/server) - Render Hyperapp statically from the server.
 - [hyperapp-transitions](https://github.com/zaceno/hyperapp-transitions) - Helps you animate transitions in your Hyperapp applications.
 
-## Resources and Others
+## Official Resources and Community
 - [Documentation](https://github.com/hyperapp/hyperapp/tree/master/docs) - Official Hyperapp documentation.
 - [Reddit](https://www.reddit.com/r/hyperapp/) - Official Hyperapp Subreddit.
 - [Slack](https://hyperappjs.herokuapp.com/) - Official Hyperapp community on Slack.
 - [Twitter](https://twitter.com/hyperappjs) - Official Hyperapp Twitter account.
+- [Codepen](https://codepen.io/hyperapp/) - Official Hyperapp Codepen account.
```
Comment (tobiasbueschel) [inline README.md]: - ~Codepen~ CodePen

File: README.md [hunk 3]
```diff
@@ -5,14 +5,16 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
...
+- [Official Resources and Community](#official-resources-and-community)
+- [Other Resources](#other-resources)
 - [Contributing](#contributing)
 
 ## Apps and Boilerplates
 
 - [deployable](https://github.com/lukejacksonn/deployable) - A ready-to-deploy static web app template built with Hyperapp.
 - [happy](https://github.com/zaceno/happy) - Happiness index calculator for teams, built with Hyperapp.
 - [hyperapp-boilerplate](https://github.com/tzellman/hyperapp-boilerplate) - Boilerplate showing how to create a production-ready Hyperapp application.
+- [hyperapp-electron](https://github.com/lukejacksonn/hyperapp-electron) - A minimal electron app starter with hyperapp, livereload and redux devtools
```
Comment (tobiasbueschel) [inline README.md]: - ~electron~ Electron
- ~hyperapp~ Hyperapp
- ~livereload~ LiveReload
- ~redux~ Redux
- ~devtools~ DevTools
- add a full stop to the end of the line

File: README.md [hunk 4]
```diff
@@ -23,13 +25,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
...
+## Official Resources and Community
 - [Documentation](https://github.com/hyperapp/hyperapp/tree/master/docs) - Official Hyperapp documentation.
 - [Reddit](https://www.reddit.com/r/hyperapp/) - Official Hyperapp Subreddit.
 - [Slack](https://hyperappjs.herokuapp.com/) - Official Hyperapp community on Slack.
 - [Twitter](https://twitter.com/hyperappjs) - Official Hyperapp Twitter account.
+- [CodePen](https://codepen.io/hyperapp/) - Official Hyperapp Codepen account.
+
+## Other Resources
+- [How Hyperapp Works](https://gist.github.com/jbucaran/8dc33b7947f3193eb2ea3d5700e27036)
+- [Pure Programming with Hyper App](https://glebbahmutov.com/blog/pure-programming-with-hyper-app/)
```
Comment (jbucaran) [inline README.md]: Hyperapp

File: README.md [hunk 5]
```diff
@@ -23,13 +25,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
...
+- [HyperApp - Tiny Library for Frontend Applications - Interview with Jorge Bucaran](https://survivejs.com/blog/hyperapp-interview/)
```
Comment (jbucaran) [inline README.md]: Hyperapp

File: README.md (hunk 3 above)
Comment (RikuVan) [reply README.md]: Thanks for the careful check. I made the requested changes. I am doubtful that LiveReload should be a proper noun in this case but no big deal, I changed it.

**Comment by RikuVan** [comment]
Thanks for the careful check. I made the requested changes. I am doubtful that LiveReload should be a proper noun in this case but no big deal, I changed it.

**Comment by jbucaran** [comment]
@RikuVan Left you a few comments, but I think it would be just faster to merge this and edit it myself. 😆

Sorry to bother you again, but can you fix & push again? 🙏

File: README.md (hunk 4 above)
Comment (RikuVan) [reply README.md]: But it feels wrong to change the name of an article (which was exactly that), even if the title is wrong? Would it be better to omit it altogether?

File: README.md (hunk 5 above)
Comment (RikuVan) [reply README.md]: Same thing this was the title of the interview, ie. that is the exact title unfortunately. Generally I think it is inappropriate to change a title name even if it has out-of-date or inaccurate usage.

Comment (RikuVan) [reply README.md]: One way to handle this is with the use of sic. Maybe I am oversensitized to such things as a former academic.

Comment (RikuVan) [reply README.md]: Maybe I could just do the following instead, omitting the title:

An Interview with Jorge Burcaran
Blog post on Hyperapp by Glen Bahmutov.

Then I am not misquoting.

File: README.md (hunk 4 above)
Comment (jbucaran) [reply README.md]: I think no one will think we are trying to be exact, but rather will end up looking sloppy. That's my opinion.

**Comment by jbucaran** [comment]
@RikuVan @tobiasbueschel Okay, let's merge it!

File: README.md [hunk 6]
```diff
@@ -23,14 +24,32 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
 - [hypernews](https://github.com/traducer/hypernews) - Hacker News clone with Hyperapp.
 
 ## Libraries and Mixins
+- [hyperapp-deepupdate](https://github.com/NoobLad/hyperapp-deepupdate) - A utility for updating deeply nested state.
+- [hyperapp-dot-notation-reducer](https://github.com/alber70g/hyperapp-dot-notation-reducer) - Allows actions to return an object with a path as a path as a property.
```
Comment (tobiasbueschel) [inline README.md]: `...with a path as a path as a property` is this intentional?

Comment (jbucaran) [reply README.md]: There is only one person that can answer that! @alber70g 😄

Comment (RikuVan) [reply README.md]: ah shoot :/ will fix. Been super busy at work the last few days if seems I am slow with this.

**Comment by RikuVan** [comment]
ok fixed the screwy dot notation description 😅

**Comment by jbucaran** [comment]
LGTM

File: README.md (hunk 6 above)
Comment (tobiasbueschel) [reply README.md]: thanks @RikuVan 🙏

**Comment by zaceno** [comment]
:tada: Great work everyone!

**Pull request merged**